client/src/rootReducer.js

```diff
@@ -3,10 +3,21 @@ import { reducer as form } from 'redux-form';
 import application from './app/appReducer';
 import auth from './auth/authReducer';
 
-export const rootReducer = combineReducers({
+const appReducer = combineReducers({
   application,
   auth,
   form,
 });
 
+export const rootReducer = (state, action) => {
+  let newState = state;
+
+  if (action.type === 'USER_LOGOUT') {
```

Comment:
It does, typically there isn't anything stateful that you want to keep around on logout and this is the easiest way to do it. If there is a need later we can adjust, but IMO it hasn't been necessary.

Fixing the import now :)